cloud/rackspace/rax_facts.py

```diff
@@ -98,6 +98,11 @@ def rax_facts(module, address, name, server_id):
         except Exception, e:
             pass
 
+    #rackspace likes to leave servers in the deleted state for a while.  
```

Comment:
We try to maintain pep8 in the rax modules.  A space following the `#` is required.